### ISSUE TITLE
Batch stress test can fail due to gas price increases

### ISSUE THREAD

**Issue by jkilpatr** [althea-net/cosmos-gravity-bridge#318]
In [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md) new governing mechanics for gas prices where introduced. These prices mandate that Ethereum blocks must have a minimum gas amount and that the amount changes based on how full recent blocks are.

Due to the way the batch stress test works it fills several blocks in a row with transactions, this this causes the gas prices to increase, if the test runner grabs the gas price before this increase occurs transactions will be rejected and the test will fail.

tl;dr dumb race condition, fix with a retry  [here](https://github.com/althea-net/cosmos-gravity-bridge/blob/main/orchestrator/test_runner/src/transaction_stress_test.rs)
